### ISSUE TITLE
Hyperparameters

### ISSUE THREAD

**Issue by mayilin0714** [FanChiMao/HWMNet#2]
I am very grateful to the author for his reply
You can refer to hyperparameters as :

LOL dataset:
Training patches: 4850 (485 x 10)
Validation: 15
Initial learning rate: 5e-5
Final learning rate: 1e-5
Training epochs: 300 (200 is enough)

MIT-5K dataset:
Training patches: 36000 (4500 x 8)
Validation: 500
Initial learning rate: 1e-4
Final learning rate: 1e-6
Training epochs: 100

**Comment by FanChiMao** [comment]
Thank you for reminding! I have added them in readme file~😊
